src/components/common/Header.js

```diff
@@ -12,7 +12,7 @@ export default function MainHeader() {
         <Menu.Item key="3">Booking</Menu.Item>
         <Menu.Item key="4">Scholarships</Menu.Item>
         <Menu.Item key="5">Contact Us (button)</Menu.Item>
-        {authState.isAuthenticated ? (
+        {/* {authState.isAuthenticated ? (
```

Comment:
this is an exception to the rule where I don't want commented out code. This will allow for a dev to come in and easily implement a logout button when Auth-0 is ready to roll.